openssl/src/ssl/mod.rs

```diff
@@ -1718,6 +1718,16 @@ impl SslContextBuilder {
         unsafe { cvt(ffi::SSL_CTX_set_num_tickets(self.as_ptr(), num_tickets)).map(|_| ()) }
     }
 
+    /// Set the context's security level, which controls the allowed parameters
+    /// and algorithms.
```

Comment:
worth adding "a number between 0 and 5"?

(The C API allows any integer and clamps to the interval [0, 5] internally, but that's not directly observable.)